ApeWorX/ape
# fix: remove unneeded github auth check when displaying available plugins

### What I did

Removed uneeded auth token check. We can get this info anon because it's public

Personal Access Tokens are only required when accessing private github information. Public repos are not private, so it makes sense that it is not needed here (https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)

### How I did it

Realized from other work and comments from @fubuloubu that it is not needed.

### How to verify it

Make sure you don't have the token set and run 

```bash
ape plugins list --all
```

You should see the available plugins

### Checklist
